developers/pymavlink.md

```diff
@@ -270,19 +270,19 @@ def set_rc_channel_pwm(id, pwm=1500):
         id (TYPE): Channel ID
         pwm (int, optional): Channel pwm value 1100-1900
     """
-    if id < 1:
+    if id < 1 or id > 18:
         print("Channel does not exist.")
         return
 
-    # We only have 8 channels
+    # Mavlink 2 supports more than 8 channels:
```

Comment:
but how many does it support ?
# .. supports up to 18 channels

Reply:
updated